src/ARCtrl/ARCtrl.fs

```diff
@@ -56,20 +56,24 @@ module ARCAux =
 [<AttachMembers>]
 type ARC(?isa : ISA.ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSystem) =
 
-    let mutable cwl = cwl
-    let mutable fs = 
+    let mutable _isa = isa
+    let mutable _cwl = cwl
+    let mutable _fs = 
         fs
         |> Option.defaultValue (FileSystem.create(FileSystemTree.Folder ("",[||])))
         |> ARCAux.updateFSByISA isa
         |> ARCAux.updateFSByCWL cwl
 
-    member val ISA : ArcInvestigation option = isa with get, set
```

Comment:
Is this change communicated Jonas? It was done to try fix an mutable issue in javascript. Please get some information AND unit test in native js from him.

Reply:
@JonasLukasczyk,
could you add a `native js` test [here](https://github.com/nfdi4plants/ARCtrl/tree/main/tests/JavaScript) which checks whether the access to the `ISA` field of the `ARC` object works as expected and it can be `mutably set`?